### ISSUE TITLE
Alternate gamut clamping methods

### ISSUE THREAD

**Issue by Artoria2e5** [Ogeon/palette#220]
The PR #219 introduces the concept of boundaries of an RGB space in another (perceptual) space. It might be interesting to use this facility to provide different ways to clamp an out-of-bounds color back into the gamut, as described in https://bottosson.github.io/posts/gamutclipping/.

**Comment by Ogeon** [comment]
Definitely! Thanks for pointing it out! I think it would be a great complement to the use of the more perceptually uniform color spaces.
